pkg/plugin/internal/client/errors.go

```diff
@@ -0,0 +1,5 @@
+package client
```

Comment:
Same here as well. Either define all custom errors here or remove this file and use `fmt.Errorf`